{"name": "Click to the func Authentication -- @1.2 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00007FF7EAA84C25+3179557]\n\t(No symbol) [0x00007FF7EA6E88A0]\n\t(No symbol) [0x00007FF7EA5791CA]\n\t(No symbol) [0x00007FF7EA5CFA67]\n\t(No symbol) [0x00007FF7EA5CFC9C]\n\t(No symbol) [0x00007FF7EA623627]\n\t(No symbol) [0x00007FF7EA5F7C6F]\n\t(No symbol) [0x00007FF7EA6202F3]\n\t(No symbol) [0x00007FF7EA5F7A03]\n\t(No symbol) [0x00007FF7EA5C06D0]\n\t(No symbol) [0x00007FF7EA5C1983]\n\tGetHandleVerifier [0x00007FF7EAAE67CD+3579853]\n\tGetHandleVerifier [0x00007FF7EAAFD1D2+3672530]\n\tGetHandleVerifier [0x00007FF7EAAF2153+3627347]\n\tGetHandleVerifier [0x00007FF7EA85092A+868650]\n\t(No symbol) [0x00007FF7EA6F2FFF]\n\t(No symbol) [0x00007FF7EA6EF4A4]\n\t(No symbol) [0x00007FF7EA6EF646]\n\t(No symbol) [0x00007FF7EA6DEAA9]\n\tBaseThreadInitThunk [0x00007FFB5656E8D7+23]\n\tRtlUserThreadStart [0x00007FFB5789BF2C+44]\n\n", "trace": "  File \"C:\\Users\\Admin\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Admin\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\herokuapp\\steps\\steps_herokuapp.py\", line 45, in step_impl\n    assert get_login_page(context).verify_subheader() == \"Welcome to the Secure Area. When you are done click logout below.\"\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\works_space\\selenium_bdd_python\\features\\herokuapp\\steps\\pages\\login_page.py\", line 46, in verify_subheader\n    return self.wait.until(EC.presence_of_element_located(self.subheader)).text # using Explicit Wait\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Admin\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 146, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "When click func Authentication", "status": "passed", "start": 1743105092279, "stop": 1743105092700}, {"name": "And input into the username \"user2\" and password \"pass2\"", "status": "passed", "start": 1743105092700, "stop": 1743105092821}, {"name": "And enter button login", "status": "passed", "start": 1743105092821, "stop": 1743105092866}, {"name": "Then Verify user login success", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00007FF7EAA84C25+3179557]\n\t(No symbol) [0x00007FF7EA6E88A0]\n\t(No symbol) [0x00007FF7EA5791CA]\n\t(No symbol) [0x00007FF7EA5CFA67]\n\t(No symbol) [0x00007FF7EA5CFC9C]\n\t(No symbol) [0x00007FF7EA623627]\n\t(No symbol) [0x00007FF7EA5F7C6F]\n\t(No symbol) [0x00007FF7EA6202F3]\n\t(No symbol) [0x00007FF7EA5F7A03]\n\t(No symbol) [0x00007FF7EA5C06D0]\n\t(No symbol) [0x00007FF7EA5C1983]\n\tGetHandleVerifier [0x00007FF7EAAE67CD+3579853]\n\tGetHandleVerifier [0x00007FF7EAAFD1D2+3672530]\n\tGetHandleVerifier [0x00007FF7EAAF2153+3627347]\n\tGetHandleVerifier [0x00007FF7EA85092A+868650]\n\t(No symbol) [0x00007FF7EA6F2FFF]\n\t(No symbol) [0x00007FF7EA6EF4A4]\n\t(No symbol) [0x00007FF7EA6EF646]\n\t(No symbol) [0x00007FF7EA6DEAA9]\n\tBaseThreadInitThunk [0x00007FFB5656E8D7+23]\n\tRtlUserThreadStart [0x00007FFB5789BF2C+44]\n\n", "trace": "  File \"C:\\Users\\Admin\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Admin\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\herokuapp\\steps\\steps_herokuapp.py\", line 45, in step_impl\n    assert get_login_page(context).verify_subheader() == \"Welcome to the Secure Area. When you are done click logout below.\"\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\works_space\\selenium_bdd_python\\features\\herokuapp\\steps\\pages\\login_page.py\", line 46, in verify_subheader\n    return self.wait.until(EC.presence_of_element_located(self.subheader)).text # using Explicit Wait\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Admin\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 146, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "attachments": [{"name": "Verify_user_login_success", "source": "a210b8e9-1593-49f2-9827-271f80750813-attachment.png", "type": "image/png"}], "start": 1743105092866, "stop": 1743105105359}], "parameters": [{"name": "username", "value": "user2"}, {"name": "password", "value": "pass2"}], "start": 1743105091919, "stop": 1743105105359, "uuid": "575ebb3d-1adc-4619-ba94-07deaa4f1cd4", "historyId": "2db698f6d7bdcd332ca13e283264a7bd", "fullName": "Example Feature: Click to the func Authentication", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Example Feature"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}